README.md

```diff
@@ -36,6 +36,43 @@ Type the name of [any `iron-*` or `paper-*` element](https://elements.polymer-pr
 </dom-module>
 ```
 
+### [pe2015] polymer element ES2015
+
+```html
+<dom-module id="${2}">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+  </template>
+  <script>
+    (function (Polymer) {
+      'use strict';
+      class ${1} {
+        beforeRegister() {
+          this.is = '${2}';
+
+          this.properties = {
+            ${3}
+          };
+        }
+
+        attached() {
+           this.async(() => {
```

Comment:
Okay, no problem, I'll take care of that. 

P.S. Is there a particular usecase for `ready`?  I'm genuinely asking I didn't mean to go on a rant. I'm just confused.